File: dayTwo/javascript/after_break/averageAndGradeOfThreeValues.js
 
function calculateAverageAndGrade(numberOne, numberTwo, numberThree){
    
    let average = (numberOne + numberTwo + numberThree) / 3;
    
    if(average >= 90){
        console.log("The average score is "+average+" and grade is A");
    }
    
    else if(average >= 80){
        console.log("The average score is "+average+" and grade is B");
    }
    
    else if(average >= 70){
        console.log("The average score is "+average+" and grade is C");
    }
    
    else if(average >= 60){
        console.log("The average score is "+average+" and grade is D");
    }
    
    else if(average < 60) {
        console.log("The average score is "+average+" and grade is F");
    }
}

calculateAverageAndGrade();
const scoreOne = 70;
const scoreTwo = 60;
const scoreThree = 50;
calculateAverageAndGrade(scoreOne, scoreTwo, scoreThree);

File: dayTwo/python/after_break/average_and_grade_of_three_values.py
 
def calculate_average_and_grade(number_one, number_two, number_three):
    
    average = (number_one + number_two + number_three) / 3
    
    if(average >= 90):
        print("The average score is ", average," and grade is A")
    
    
    elif(average >= 80):
        print("The average score is ", average," and grade is B")
    
    
    elif(average >= 70):
        print("The average score is ", average," and grade is C")
    
    
    elif(average >= 60):
        print("The average score is ", average," and grade is D")

    
    elif(average < 60):
        print("The average score is ", average," and grade is F")
    

score_one = 70
score_two = 60
score_three = 50
calculate_average_and_grade(score_one, score_two, score_three)
